BaseTools/Plugin/OverrideValidation/OverrideValidation.py

```diff
@@ -8,14 +8,14 @@
 ##
 
 
+import argparse

```

Comment:
Is it valuable to maintain the history of this commit? If we just see this as part of the override validation plugin, we could collapse this with the commit adding it.

Reply:
Oh, I see your comment now on the PR that these are two core commits. Resolving this.